core/src/main/java/org/apache/calcite/rel/rel2sql/SqlImplementor.java

```diff
@@ -390,6 +390,20 @@ public static SqlNode convertConditionToSqlNode(RexNode node,
       joinContext =
           leftContext.implementor().joinContext(leftContext, rightContext);
       return joinContext.toSql(null, node);
+    case IS_NOT_TRUE:
+    case IS_TRUE:
+      if (!dialect.getConformance().allowIsTrue()) {
+        SqlOperator op1 = dialect.getTargetFunc((RexCall) node);
```

Comment:
change its name to something descriptive like `targetFunc`

Reply:
Hi, @nosheenSheikh here if you see this statement is just fetching the operator from the node. so that is the reason I have mentioned its name as **op1**

Reply:
@nosheenSheikh it's done please check.